core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -653,6 +653,19 @@ class ZkUtils(val zkClient: ZkClient,
     (dataStr, stat)
   }
 
+  def readData(path: String, returnNoneIfPathNotExists: Boolean): Option[String] = {
```

Comment:
Okey, I just wanted to ask why shouldn't we use the readDataMaybeNull because as I see every other class uses that, but with this explanation it makes sense. I will create a separate ticket for the refactor.